workflows/rules/common.smk

```diff
@@ -42,56 +42,43 @@ METADATA_DIR = datapaths.get_metadata_dir()
 # To understand the level of data, please refere to PyCytominer documentation
 # https://github.com/cytomining/pycytominer
 
-# Level 2 data: converted into parquet format
-CYTOTABLE_CONVERTED_PLATE_DATA = datapaths.build_path(
-    data_type="plate_data", use_converted=True
-)
-CYTOTABLE_CONVERTED_PLATE_DATA_EXTENDED = expand(
-    CYTOTABLE_CONVERTED_PLATE_DATA, basename=plate_name
-)
-
-# level 2.5 data: annotated level 2 data based on given metadata (e.g treatments)
-ANNOTATED_DATA = datapaths.build_path(data_type="annotated")
-ANNOTATED_DATA_EXPAND = expand(ANNOTATED_DATA, basename=plate_name)
-
-# level 3 data: aggregated profile based on given aggregation level
-# (e.g aggregating single-cell data to the well level)
-AGGREGATE_DATA = datapaths.build_path(data_type="aggregated")
-AGGREGATE_DATA_EXPAND = expand(AGGREGATE_DATA, basename=plate_name)
-
-# level 4a data: noramlzied profile
-NORMALIZED_DATA = datapaths.build_path(data_type="normalized")
-NORMALIZED_DATA_EXPAND = expand(NORMALIZED_DATA, basename=plate_name)
-
-# level 4b: selected features profile
-SELECTED_FEATURE_DATA = datapaths.build_path(data_type="feature_select")
-SELECTED_FEATURE_DATA_EXPAND = expand(SELECTED_FEATURE_DATA, basename=plate_name)
-
-# level 5: Consensus profile captures unique signatures that resulted from
-# any external factor (e.g pertubations)
-CONSENSUS_DATA = datapaths.build_path(data_type="consensus")
-CONSENSUS_DATA_EXPAND = expand(CONSENSUS_DATA, basename=plate_name)
-
-# other outputsa
-# Cell counts: cell counts per well in level 2 data
-CELL_COUNTS = datapaths.build_path(data_type="consensus")
-CELL_COUNTS_EXPANDED = expand(CELL_COUNTS, basename=plate_name)
-
-
-def get_input(
-    data_type: str,
+def get_data_path(
+    input_type: str,
     use_converted: Optional[bool] = False,
     tolist: Optional[bool] = False,
 ) -> str:
-    """Returns absolute path of an input
+    """Returns absolute path of an input.
+
+    The supported input types are:
+
+    plate_data (level 2):
```

Comment:
Consider adding details covering "levels" to formal documentation for this project in a relevant location (if it isn't already).

Reply:
I have provided a link to `pycytominer` package that provides explanations of the `levels` of datasets.